pkg/visor/api.go

```diff
@@ -540,12 +540,15 @@ func (v *Visor) GetAppError(appName string) (string, error) {
 
 // GetAppConnectionsSummary implements API.
 func (v *Visor) GetAppConnectionsSummary(appName string) ([]appserver.ConnectionSummary, error) {
-	cSummary, err := v.procM.ConnectionsSummary(appName)
-	if err != nil {
-		return nil, err
-	}
+	if v.procM != nil {
+		cSummary, err := v.procM.ConnectionsSummary(appName)
+		if err != nil {
+			return nil, err
+		}
 
-	return cSummary, nil
+		return cSummary, nil
+	}
+	return nil, nil
```

Comment:
Should we return an error here if there is no `procM`?

Reply:
Actually no need for error. `procM` just sometimes start/initilaize with delay, and this is just checking for that, also if not start at all then visor not running with this error:
```
[2022-05-11T14:45:30+04:30] ERROR [visor:startup]: initializing module launcher returning error: lol
[2022-05-11T14:45:30+04:30] ERROR []: Failed to start visor.
```